python_modules/dagster/dagster/core/execution/api.py

```diff
@@ -134,7 +135,7 @@ def gen_ignore_duplicate_run_worker():
                 pipeline_run=pipeline_run,
                 instance=instance,
                 run_config=pipeline_run.run_config,
-                raise_on_error=False,
+                raise_on_error=raise_on_error,
```

Comment:
from here on out this is relying on pre-existing behavior / a pre-existing 'raise_on_error' flag